### PR TITLE
docs(blog): Gatsby Gazette: The 2019 Recap

### PR DESCRIPTION
**Pull request by sidharthachatterjee**

### CODE REVIEW

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+(https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.120...gatsby@2.1.0)
+
+The `useStaticQuery` hook was the first minor release in 2019. The author of this post built this just in time for the first stable release of React Hooks.
+
+### 2.2.0: Schema Customisation
+
+(2019-03-19)
+(https://github.com/gatsbyjs/gatsby/compare/gatsby@2.1.39...gatsby@2.2.0)
+
+This was a huge milestone in the development of Gatsby's GraphQL data layer. Built by Mikhail and Stefan, this enabled users to declare their own custom GraphQL schema. Before this, you site's schema was exclusively inferred by Gatsby leading to a dependency on data and making it brittle.
```
Comment (LekoArts) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]: Not commenting every occurrence so valid for whole document:

Should we link the names to GitHub/Twitter profiles? As cool as e.g. Mikhail and Stefan are, they are not _that_ famous yet.

Comment (sidharthachatterjee) [reply docs/blog/2020-02-18-gazette-2019-recap/index.md]: Haha, yup. Need to link folks to Twitter and feature titles to those diff urls

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+### 2.3.0: Telemetry Instrumentation
+
+(2019-03-26)
+(https://github.com/gatsbyjs/gatsby/compare/gatsby@2.2.13...gatsby@2.3.0)
+
+2.3.0 marked the release of telemetry in Gatsby. Built by Jarmo, Telemetry is anonymous analytics data that we send back to Gatsby to better understand the way our users interact with Gatsby CLI. This makes common errors _visible_ to the team and enables them to fix them even sooner.
```
Comment (calcsam) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]: Need last names and links to GH or Twitter

Comment (sidharthachatterjee) [reply docs/blog/2020-02-18-gazette-2019-recap/index.md]: Yeah, those are coming

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+1348 folks started contributing to Gatsby in 2019.
+
+Together, we made 5657 commits to Gatsby in 2019.
+
+That's a 29% increase from 2018 (4364) and a 153% increase from 2017 (2236).
+
+We released Gatsby 467 times in 2019. 😱 (Run `npm view gatsby time` to display the full list)
+
+Out of these, 18 were minor releases.
```
Comment (calcsam) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]:
```suggestion
Out of these, 18 were minor releases adding significant functionality to the Gatsby project we want to highlight below:
```

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+(2019-11-19)
+(https://github.com/gatsbyjs/gatsby/compare/gatsby@2.17.17...gatsby@2.18.0)
+
+Vlad (who recently joined the Gatsby team) killed it in his first month (😱) and added support for automatic Schema rebuilding in the Gatsby CLI. This previously required users to restart the gatsby CLI when changing the schema of their gatsby sites.
+
+There’s a _lot_ more that’s not included here including error message upgrades, several performance improvements, React prerelease tests, an in progress TS migration and more.
+
+## 2020
+
+We started off this year on a solid note with the release of Gatsby Builds, the fastest way to build your Gatsby sites in the cloud. In open source land, we've done plenty already as well.
```
Comment (LekoArts) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]: Link to the release blog post here :D

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+
+- making the CLI faster and cleaner
+- adding tools for consuming themes easily
+- making switching data sources for your site even easier
+- better error messaging
+- scaling to hundreds of thousands of pages
+
+We'll tell you more as we get closer to these.
+
+I hope you've enjoyed reading this as much as I've enjoyed writing it.
```
Comment (calcsam) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]: Probably leave off the last couple lines -- they read more like an email than a blog post :)

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+
+We started off this year on a solid note with the release of Gatsby Builds, the fastest way to build your Gatsby sites in the cloud. In open source land, we've done plenty already as well.
+
+### Internationalizing Gatsby Docs
+
+Nat resumed her effort of helping internationalize the Gatsby documentation. We want Gatsby to be the most accessible way to build for the web. If you're interested in contributing translations in a language of your choice, go check out the translation contribution guidelines!
+
+### Accessible routing
+
+Madalyn released accessible client side routing in Gatsby in 2.19.8. These improvements enable people relying on screen readers to successfully navigate sites built with Gatsby.
```
Comment (LekoArts) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]: Also link to blog post here

**Comment by calcsam** [comment]
This is awesome @sidharthachatterjee -- glad you wrote this!!!

File: docs/blog/2020-02-18-gazette-2019-recap/index.md (hunk 5 above)
Comment (sidharthachatterjee) [reply docs/blog/2020-02-18-gazette-2019-recap/index.md]: Yeah, I felt like that too! 😅

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+(https://github.com/gatsbyjs/gatsby/compare/gatsby@2.14.7...gatsby@2.15.0)
+
+Another big one! Mikhail rewrote much of Gatsby's Node model to support arbitrary data stores in the future. This was done to enable faster data stores in the future and larger Gatsby sites.
+
+### 2.16.0: Structured logging
+
+(2019-10-14)
+(https://github.com/gatsbyjs/gatsby/compare/gatsby@2.15.37...gatsby@2.16.0)
+
+Michal and Sid released their work on the Structured logging effort in this release. Most of this was a ground up rewrite of the way Gatsby stores and prints logs and activities enabling alternative UIs for the Gatsby CLI including a Gatsby Desktop app that is coming _soon_ :wink:
```
[the remark this answers is not in the document]
Comment (sidharthachatterjee) [reply docs/blog/2020-02-18-gazette-2019-recap/index.md]: Good point. Changed it to say _in the future_

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+
+For our more frequent readers who might already know much of this, we'll follow up with a walkthrough of our plans for 2020.
+
+Before we dive in, thank you, dear reader. If you've been contributing code to Gatsby, helping improve our documentation, reporting bugs and issues or even simply using it, you've enabled so much of the progress we've made and will continue to make this year. Our superpower is _you_, the Gatsby community.
+
+## The 2019 Recap
+
+Let's get into it then.
+
+1348 folks started contributing to Gatsby in 2019.
```
Comment (pvdz) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]: Is this "new" or "at least once in 2019"? (I'm curious, no need to change it on my behalf)

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+
+Together, we made 5657 commits to Gatsby in 2019.
+
+That's a 29% increase from 2018 (4364) and a 153% increase from 2017 (2236).
```
Comment (pvdz) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]: I don't think this needs a line break? (If this was a list I'd expect bullet points instead)

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+
+There’s a _lot_ more that’s not included here including error message upgrades, several performance improvements, React prerelease tests, an in progress TS migration and more.
+
+## 2020
+
+We started off this year on a solid note with the release of [Gatsby Builds](https://www.gatsbyjs.org/blog/2020-01-27-announcing-gatsby-builds-and-reports/), the _fastest_ way to build your Gatsby sites in the cloud. In OSS land, we've done plenty already as well.
+
+### Internationalizing Gatsby Docs
+
+Nat resumed her effort of helping internationalize the Gatsby documentation. We want Gatsby to be the most accessible way to build for the web. If you're interested in contributing translations in a language of your choice, go check out the translation contribution guidelines!
```
Comment (pvdz) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]: Seems a little inconsistent to not use twitter links here, but ok

Comment (sidharthachatterjee) [reply docs/blog/2020-02-18-gazette-2019-recap/index.md]: Oops, I totally missed this one. Adding!

File: docs/blog/2020-02-18-gazette-2019-recap/index.md (hunk 9 above)
Comment (sidharthachatterjee) [reply docs/blog/2020-02-18-gazette-2019-recap/index.md]: Good point

File: docs/blog/2020-02-18-gazette-2019-recap/index.md (hunk 8 above)
Comment (sidharthachatterjee) [reply docs/blog/2020-02-18-gazette-2019-recap/index.md]: New people who contributed for the first time in 2019

Found folks of all time and folks in 2019 and did a Set difference

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 11]
```diff
@@ -0,0 +1,175 @@
...
+
+#### [2019-02-13](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.120...gatsby@2.1.0)
+
+The `useStaticQuery` hook was the first minor release in 2019. [Sid Chatterjee](https://twitter.com/chatsidhartha) built this just in time for the first stable release of React Hooks.
+
+### 2.2.0: Schema Customization
+
+#### [2019-03-19](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.1.39...gatsby@2.2.0)
+
+This was a huge milestone in the development of Gatsby's GraphQL data layer. Built by [Mikhail Novikov](https://twitter.com/freiksenet) and [Stefan Probst](https://github.com/stefanprobst), this enabled users to declare their own custom GraphQL schema. Before this, your site's schema was exclusively inferred by Gatsby leading to a dependency on data and making it brittle. Read more about [Schema Customization](https://www.gatsbyjs.org/blog/2019-03-18-releasing-new-schema-customization/) in the release blog post.
```
Comment (laurieontech) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]:
```suggestion
This was a huge milestone in the development of Gatsby's GraphQL data layer. Built by [Mikhail Novikov](https://twitter.com/freiksenet) and [Stefan Probst](https://github.com/stefanprobst), this enabled users to declare their own custom GraphQL schema. Before this, your site's schema was exclusively inferred by Gatsby leading to a dependency on data and making it brittle. Read more about [Schema Customization](/blog/2019-03-18-releasing-new-schema-customization/) in the release blog post.
```

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 12]
```diff
@@ -0,0 +1,175 @@
...
+
+### 2.3.0: Telemetry
+
+#### [2019-03-26](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.2.13...gatsby@2.3.0)
+
+Gatsby 2.3.0 marked the release of telemetry in Gatsby. Built by [Jarmo Isotalo](https://twitter.com/JarmoIsotalo), Telemetry is [anonymous analytics data](https://www.gatsbyjs.org/docs/telemetry/) that we send back to Gatsby to better understand the way our users interact with Gatsby CLI. This makes common errors _visible_ to the team and enables them to fix them even sooner.
```
Comment (laurieontech) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]:
```suggestion
Gatsby 2.3.0 marked the release of telemetry in Gatsby. Built by [Jarmo Isotalo](https://twitter.com/JarmoIsotalo), Telemetry is [anonymous analytics data](/docs/telemetry/) that we send back to Gatsby to better understand the way our users interact with Gatsby CLI. This makes common errors _visible_ to the team and enables them to fix them even sooner.
```

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 13]
```diff
@@ -0,0 +1,175 @@
...
+
+#### [2019-05-31](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.7.6...gatsby@2.8.0)
+
+Another release by [Stefan Probst](https://github.com/stefanprobst), this added support for sorting and filtering for resolved fields (ones that are _not_ on the Node itself). This enables users to filter and sort on fields that link to other Nodes, for example `frontmatter.author.name` when author comes from a linked Node.
+
+### 2.9.0: Per Page Manifest
+
+#### [2019-06-11](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.8.8...gatsby@2.9.0)
+
+This release by [Anthony Marcar](https://github.com/moocar) was a _fundamental_ change to the Gatsby runtime, changing how page data is stored and fetched. Not only did this enable building larger Gatsby sites than before (we're talking in the order of thousands of pages) but also reduced the overhead of loading Gatsby pages from a linearly growing metric to constant. Read more about this in the [Per Page Manifest release blog post](https://www.gatsbyjs.org/blog/2019-06-12-performance-improvements-for-large-sites/).
```
Comment (laurieontech) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]:
```suggestion
This release by [Anthony Marcar](https://github.com/moocar) was a _fundamental_ change to the Gatsby runtime, changing how page data is stored and fetched. Not only did this enable building larger Gatsby sites than before (we're talking in the order of thousands of pages) but also reduced the overhead of loading Gatsby pages from a linearly growing metric to constant. Read more about this in the [Per Page Manifest release blog post](/blog/2019-06-12-performance-improvements-for-large-sites/).
```

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 14]
```diff
@@ -0,0 +1,175 @@
...
+
+#### [2019-11-19](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.17.17...gatsby@2.18.0)
+
+[Vladimir Razuvaev](https://twitter.com/vrazuvaev) (who recently joined the Gatsby team) killed it in his first month (😱) and added support for automatic Schema rebuilding in the Gatsby CLI. This previously required users to restart the gatsby CLI when changing the schema of their gatsby sites.
+
+There’s a _lot_ more that’s not included here including error message upgrades, several performance improvements, React prerelease tests, an in progress TS migration and more.
+
+## 2020
+
+We started off this year on a solid note with the release of [Gatsby Builds](https://www.gatsbyjs.org/blog/2020-01-27-announcing-gatsby-builds-and-reports/), the _fastest_ way to build your Gatsby sites in the cloud. In OSS land, we've done plenty already as well.
```
Comment (laurieontech) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]:
```suggestion
We started off this year on a solid note with the release of [Gatsby Builds](/blog/2020-01-27-announcing-gatsby-builds-and-reports/), the _fastest_ way to build your Gatsby sites in the cloud. In OSS land, we've done plenty already as well.
```

File: docs/blog/2020-02-18-gazette-2019-recap/index.md [hunk 15]
```diff
@@ -0,0 +1,175 @@
...
+
+### Internationalizing Gatsby Docs
+
+[Nat Alison](https://twitter.com/tesseralis) resumed her effort of helping internationalize the Gatsby documentation. We want Gatsby to be the most accessible way to build for the web. If you're interested in contributing translations in a language of your choice, go check out the translation contribution guidelines!
+
+### Accessible routing
+
+[Madalyn Parker](https://twitter.com/madalynrose) released [Accessible Client Side Routing](https://www.gatsbyjs.org/blog/2020-02-10-accessible-client-side-routing-improvements/) in Gatsby in 2.19.8. These improvements enable people relying on screen readers to successfully navigate sites built with Gatsby.
```
Comment (laurieontech) [inline docs/blog/2020-02-18-gazette-2019-recap/index.md]:
```suggestion
[Madalyn Parker](https://twitter.com/madalynrose) released [Accessible Client Side Routing](/blog/2020-02-10-accessible-client-side-routing-improvements/) in Gatsby in 2.19.8. These improvements enable people relying on screen readers to successfully navigate sites built with Gatsby.
```

**Pull request merged**